Pig Stack Trace
---------------
ERROR 6000: 
<line 3, column 0> Output Location Validation Failed for: 'hdfs://localhost.localdomain:8020/user/cloudera/outputforincome More info to follow:
Output directory hdfs://localhost.localdomain:8020/user/cloudera/outputforincome already exists

org.apache.pig.impl.plan.VisitorException: ERROR 6000: 
<line 3, column 0> Output Location Validation Failed for: 'hdfs://localhost.localdomain:8020/user/cloudera/outputforincome More info to follow:
Output directory hdfs://localhost.localdomain:8020/user/cloudera/outputforincome already exists
	at org.apache.pig.newplan.logical.rules.InputOutputFileValidator$InputOutputFileVisitor.visit(InputOutputFileValidator.java:95)
	at org.apache.pig.newplan.logical.relational.LOStore.accept(LOStore.java:66)
	at org.apache.pig.newplan.DepthFirstWalker.depthFirst(DepthFirstWalker.java:64)
	at org.apache.pig.newplan.DepthFirstWalker.depthFirst(DepthFirstWalker.java:66)
	at org.apache.pig.newplan.DepthFirstWalker.depthFirst(DepthFirstWalker.java:66)
	at org.apache.pig.newplan.DepthFirstWalker.depthFirst(DepthFirstWalker.java:66)
	at org.apache.pig.newplan.DepthFirstWalker.walk(DepthFirstWalker.java:53)
	at org.apache.pig.newplan.PlanVisitor.visit(PlanVisitor.java:52)
	at org.apache.pig.newplan.logical.rules.InputOutputFileValidator.validate(InputOutputFileValidator.java:45)
	at org.apache.pig.backend.hadoop.executionengine.HExecutionEngine.compile(HExecutionEngine.java:285)
	at org.apache.pig.PigServer.compilePp(PigServer.java:1324)
	at org.apache.pig.PigServer.executeCompiledLogicalPlan(PigServer.java:1249)
	at org.apache.pig.PigServer.execute(PigServer.java:1241)
	at org.apache.pig.PigServer.executeBatch(PigServer.java:335)
	at org.apache.pig.tools.grunt.GruntParser.executeBatch(GruntParser.java:137)
	at org.apache.pig.tools.grunt.GruntParser.processScript(GruntParser.java:475)
	at org.apache.pig.tools.pigscript.parser.PigScriptParser.Script(PigScriptParser.java:804)
	at org.apache.pig.tools.pigscript.parser.PigScriptParser.parse(PigScriptParser.java:449)
	at org.apache.pig.tools.grunt.GruntParser.parseStopOnError(GruntParser.java:194)
	at org.apache.pig.tools.grunt.GruntParser.parseStopOnError(GruntParser.java:170)
	at org.apache.pig.tools.grunt.Grunt.run(Grunt.java:69)
	at org.apache.pig.Main.run(Main.java:538)
	at org.apache.pig.Main.main(Main.java:157)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.apache.hadoop.util.RunJar.main(RunJar.java:208)
Caused by: org.apache.hadoop.mapred.FileAlreadyExistsException: Output directory hdfs://localhost.localdomain:8020/user/cloudera/outputforincome already exists
	at org.apache.hadoop.mapreduce.lib.output.FileOutputFormat.checkOutputSpecs(FileOutputFormat.java:132)
	at org.apache.pig.newplan.logical.rules.InputOutputFileValidator$InputOutputFileVisitor.visit(InputOutputFileValidator.java:80)
	... 27 more
================================================================================
